trestle/common/const.py

```diff
@@ -228,7 +228,7 @@
 # need to strip white space from end of value
 PROPERTY_REGEX = r'(?:###\s+Property\s+)([^\s]*)\s*:\s+(.*)'
 
-PART_REGEX = r'(?:##\s+Part\s+)(.*)'
+PART_REGEX = r'(?:##\s+part\s+)(.*)'
 
```

Comment:
I think it's worth changing this to have `(?i)` in front to indicate directly that case is ignored when matching the string.  Then you can remove all the `lower()` references in the code that do matching.  Otherwise if a `lower()` is omitted it could be a bug.

Reply:
Good suggestion, didn't think of modifying regex. Will adjust